pkg/commands/app.go

```diff
@@ -1199,7 +1242,8 @@ func showVersion(cacheDir, outputFormat string, w io.Writer) error {
 }
 
 func validateArgs(cmd *cobra.Command, args []string) error {
-	// '--clear-cache', '--download-db-only', '--download-java-db-only', '--reset', '--reset-checks-bundle' and '--generate-default-config' don't conduct the subsequent scanning
+	// '--clear-cache' (removed), '--download-db-only' (removed), '--download-java-db-only', '--reset',
```

Comment:
nit:
```suggestion
	// '--clear-cache' (removed), '--download-db-only' (removed), '--download-java-db-only', '--reset' (removed),
```

Reply:
Fixed in https://github.com/aquasecurity/trivy/pull/6993/commits/01f901c93fc30009d0b57ac28908b6aa0442885e